### PR TITLE
Reformat client docs

### PR DESCRIPTION
**Pull request by ZuzaOsa**
Remove html tags from documentation, replace with markdown. ``Client``, ``ClientOptions`` and ``QueryOptions`` were updated.

### CODE REVIEW

**Comment by adespawn** [comment]
We need some automatic way of building and sharing documentation, but that's a separate issue...

**Comment by adespawn** [comment]
In the meantime can you share before and after (ex specific HTML files) to speed up a review?

**Comment by ZuzaOsa** [comment]
> In the meantime can you share before and after (ex specific HTML files) to speed up a review?

Unfortunately the HTMLs do not display properly if they are opened separately from the other files. Probably the easiest solution is to generate docs from ``simple-query`` and from this PR to compare them locally.

**Comment by wprzytula** [comment]
> We need some automatic way of building and sharing documentation, but that's a separate issue...

Take a look at docs auto deployment in Rust driver:
- https://github.com/scylladb/scylla-rust-driver/blob/main/.github/workflows/docs-pages.yaml
- https://github.com/scylladb/scylla-rust-driver/blob/main/.github/workflows/docs-pr.yaml

File: lib/client-options.js [hunk 1]
```diff
@@ -10,35 +10,38 @@ const auth = require("./auth");
...
+ *
  * @property {String} [localDataCenter] The local data center to use.
- * <p>
- *   If using DCAwareRoundRobinPolicy (default), this option is required and only hosts from this data center are
- *   connected to and used in query plans.
- * </p>
+ *
+ * If using DCAwareRoundRobinPolicy (default), this option is required and only hosts from this data center are
+ * connected to and used in query plans.
+ *
```
Comment (adespawn) [inline lib/client-options.js:35]: It's my mistake, but this field is missing TODO. Only ``contactPoints`` is somehow supported

**Review by adespawn — APPROVED** [review]

File: lib/client-options.js (hunk 1 above)
Comment (ZuzaOsa) [reply lib/client-options.js:35]: Ok, I fixed that.

**Pull request merged**